Source/WebCore/CMakeLists.txt

```diff
@@ -1727,6 +1727,7 @@ set(WebCore_USER_AGENT_STYLE_SHEETS
     ${WEBCORE_DIR}/css/plugIns.css
     ${WEBCORE_DIR}/css/quirks.css
     ${WEBCORE_DIR}/css/svg.css
+    ${WEBCORE_DIR}/css/horizontalFormControls.css
```

Comment:
Nit: keep this sorted.

And did you forget to add this file?

Reply:
Fixed.